Hummingbird.docc/Articles/EncodingAndDecoding.md

```diff
@@ -60,11 +62,11 @@ router.post("user") { request, context -> HTTPResponse.Status in
     return .ok
 }
 ```
-Like the standard `Decoder.decode` functions `Request.decode` can throw an error if decoding fails. The decode function is also async as the request body is an asynchronous sequence of `ByteBuffers`. We need to collate the request body into one buffer before we can decode it.
+Like the standard ``Decoder.decode`` functions ``Request.decode`` can throw an error if decoding fails. The decode function is also async as the request body is an asynchronous sequence of `ByteBuffer`s. We need to collate the request body into one buffer before we can decode it.
```

Comment:
I prefer when linking to symbols in docs, that the first instance of the symbol gets a link and subsequent instances don't. Otherwise the docs become a mess of blue symbols and they are difficult to read.

Also you need the fully qualified name for function name links